CHANGELOG.md

```diff
@@ -1555,7 +1555,7 @@ replace github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8
 
 ### Improvements
 
-* (deps) Migrate to [CometBFT](https://github.com/cometbft/cometbft). Follow the instructions in the [release notes](./RELEASE_NOTES.md).
+* (deps) Migrate to [CometBFT](https://github.com/cometbft/cometbft). Follow the instructions in the [release process](./RELEASE_PROCESS.md).
```

Comment:
revert

Reply:
`./RELEASE_NOTES.md` is no longer a valid file

Reply:
You can get it from the tag of the version the changelog is about or directly link to the proper release, like https://github.com/cosmos/cosmos-sdk/releases/tag/v0.46.3